deployer/scripts/install.sh

```diff
@@ -373,6 +373,10 @@ function generate_objects() {
   generate_kibana
   generate_curator
   generate_fluentd
+
```

Comment:
should that also be removed from https://github.com/ewolinetz/origin-aggregated-logging/blob/644af15b3d1722be7df576136917963616194373/deployer/scripts/upgrade.sh#L431 ?